internal/test/e2e/events_test.go

```diff
@@ -21,6 +21,7 @@ import (
 func TestEvents(t *testing.T) {
 	// list all webhooks
 	allEvents := []func(string, map[string]string) webhooks.Webhook{
+		api.WebhookACLUpdate,
```

Comment:
This PR looks good but 2 things would be nice to have 

1. A test where we manually form a few contracts, upload+download, block, upload+download (fails), unblock, upload+download
2. Enable `TestBlocklist` again. Seems like that one slipped through the cracks at some point